webapp/src/main/java/org/opentestsystem/rdw/admin/security/User.java

```diff
@@ -1,32 +1,36 @@
 package org.opentestsystem.rdw.admin.security;
 
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
+import org.opentestsystem.rdw.admin.common.security.PermissionScope;
 import org.springframework.security.core.GrantedAuthority;
-import org.springframework.security.core.userdetails.UserDetails;
 
 import java.util.Collection;
 import java.util.Map;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
 
-public class User implements UserDetails {
+public class User extends org.springframework.security.core.userdetails.User {
 
     private String id;
     private String email;
     private String firstName;
     private String lastName;
     private Map<String, Permission> permissionsById;
-    private String username;
-    private String password;
-    private Collection<? extends GrantedAuthority> authorities;
-    private boolean accountNonExpired;
-    private boolean accountNonLocked;
-    private boolean credentialsNonExpired;
-    private boolean enabled;
+    private Map<Long, GroupGrant> groupsById;
```

Comment:
I ported the RDW_Reporting User changes over.  I don't believe RDW_Admin populates or uses the groupsById property in the hotfix, but I'm a fan of consistency.